socialhome/streams/app/tests/fixtures/jsonContext.fixtures.js

```diff
@@ -73,4 +75,4 @@ const getContext = function (args = {}) {
     })
 }
 
-export {getFakeAuthor, getFakePost, getProfile, getContext}
+export {getFakeAuthor, getFakeContent, getProfile, getContext}
```

Comment:
At some point, we'll need to generate fixtures from Django models directly. It avoids false-positives when the JSON changes.